app/services/foreman/foreman_url_renderer.rb

```diff
@@ -16,18 +16,15 @@ def default_url_options
 
     apipie :method, 'Returns Foreman unattended URL' do
       desc 'Returns URL to Foreman or Smart Proxy depending on host.subnet.template proxy configuration.'
-      # optional object_of: String, desc: 'template kind (provision, script, ...)'
+      required :action, String, desc: 'template kind (provision, script, ...)'
       # optional object_of: Hash, desc: 'URL parameters, use key named :unencoded for parameters which must not be URL-encoded'
```

Comment:
Sorry for bothering you, but since you're touching this... :) Could we document the whole method this time? Somehow we missed that :/

Just so it's easier to use DSL:
```ruby
required :action, String, desc: 'Template kind ("provision", "script", ...)'
optional :params, Hash, desc: '...'
optional :unescaped_params, Hash, desc: '...'
returns String, desc: 'Rendered URL'
example '<%= foreman_url("provision") %> #=> "https://foreman.example.com:8443/unattended/provision?token=buildToken"'
```
:cookie: If could come up with more sophisticated examples :)